### DISCUSSION TITLE
Add support for WEC TV

### DISCUSSION THREAD

**Discussion by JustBeALlama** [f1multiviewer/issue-tracker#397] [Feature requests] [answered]
With the addition of IndyCar and soon NASCAR, a possibility for WEC should be possible to implement as it also provides several onboards and main feeds just like F1TV has.

**Comment by f1multiviewer** [comment, marked as answer]
Done :)

**Reply by LQDB** [reply]
I would like to have the login to FIA WEC.tv on a new window (on the browser) to enable me to use my Apple ID to log in, as I can't use it through your app login page.

<img width="604" alt="Screenshot 2024-06-15 at 9 48 26 PM" src="https://github.com/f1multiviewer/issue-tracker/assets/136699876/8ba13852-d908-4f8f-abfb-90e9a63b14f4">
